examples/minimqtt_simpletest.py

```diff
@@ -1,30 +1,29 @@
 # SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
 # SPDX-License-Identifier: MIT
 
+import os
 import ssl
 import socketpool
 import wifi
 import adafruit_minimqtt.adafruit_minimqtt as MQTT
 
-# Add a secrets.py to your filesystem that has a dictionary called secrets with "ssid" and
-# "password" keys with your WiFi credentials. DO NOT share that file or commit it into Git or other
+# Add settings.toml to your filesystem CIRCUITPY_WIFI_SSID and CIRCUITPY_WIFI_PASSWORD keys
+# with your WiFi credentials. DO NOT share that file or commit it into Git or other
 # source control.
-# pylint: disable=no-name-in-module,wrong-import-order
-try:
-    from secrets import secrets
-except ImportError:
-    print("WiFi secrets are kept in secrets.py, please add them there!")
-    raise
-
-# Set your Adafruit IO Username and Key in secrets.py
+
+# Set your Adafruit IO Username and Key in settings.toml
 # (visit io.adafruit.com if you need to create an account,
 # or if you need your Adafruit IO key.)
-aio_username = secrets["aio_username"]
-aio_key = secrets["aio_key"]
-
-print("Connecting to %s" % secrets["ssid"])
-wifi.radio.connect(secrets["ssid"], secrets["password"])
-print("Connected to %s!" % secrets["ssid"])
+aio_username = os.getenv("aio_username")
+aio_key = os.getenv("aio_key")
+aio_broker = os.getenv("broker")
+aio_port = os.getenv("port")
```

Comment:
@BlitzCityDIY  Do we specify in any guides that `aio_port` and `aio_broker` are required in the TOML file? The comments on L14 specifies only IO username and password are set within this file.

I'm open to defining the MQTT client like below to make it more transparent to the user of our MQTT client that a) it's not IO-specific and b) where exactly this sketch is connecting to:

```
mqtt_client = MQTT.MQTT(
    broker="io.adafruit.com",
    port=8883,
    username=os.getenv("CIRCUITPY_WIFI_SSID"),
    password=os.getenv("CIRCUITPY_WIFI_PASSWORD"),
    socket_pool=pool,
    ssl_context=ssl.create_default_context(),
)
```

Reply:
I agree, I think that makes sense. I was mirroring the secrets syntax but having it be defined like that will be easier for folks to understand I think.